src/bin/tupm/ui.rs

```diff
@@ -314,29 +314,16 @@ impl AccountEditView {
         };
 
         for field in FIELDS.into_iter() {
-            let id = format!("{}_{}", VIEW_ID_EDIT, field.name);
             let mut edit_view = EditView::new();
             edit_view.set_secret(field.secret);
+            let width = SizeConstraint::AtLeast(30);
+            let label = labelify(field.name);
             if !field.multiline {
-                v_layout.add_child(
-                    LinearLayout::horizontal()
-                        .child(TextView::new(labelify(field.name)))
-                        .child(BoxView::new(
-                            SizeConstraint::AtLeast(30),
-                            SizeConstraint::AtMost(1),
-                            edit_view.with_id(id),
-                        )),
-                );
+                let height = SizeConstraint::AtMost(1);
+                AccountEditView::add_field(&mut v_layout, edit_view, width, height, field, label);
             } else {
-                v_layout.add_child(
-                    LinearLayout::vertical()
-                        .child(TextView::new(labelify(field.name)))
-                        .child(BoxView::new(
-                            SizeConstraint::AtLeast(30),
-                            SizeConstraint::Fixed(10),
-                            TextArea::new().with_id(id),
-                        )),
-                );
+                let height = SizeConstraint::Fixed(10);
+                AccountEditView::add_field(&mut v_layout, edit_view, width, height, field, label);
```

Comment:
In principle, the intended refactoring is fine. (Even though the amount of code duplication it is addressing is very small.) However, this particular change results in negative consequences for the Account Edit UI. The single-line fields are no longer placed to the right of their labels, but instead on a separate line below the labels, leading to a more cluttered appearance. More importantly, the multi-line "notes" field is now single-line and incapable of editing multi-line notes.

Reply:
that is really weird, it shows no change in my environment.

I will delete my fork and redo the fork. That should give me a clean base to work on. I strongly suspect our repo's are not equal and that causes differences in result.

Reply:
I tested with 97682c3e04372d0b8846ad30a6fea6ff0a6d9c1f, which (at this time) is what your `experiment_refactoring` branch is pointing to. If we're both using 97682c3, we should have the same code and should see the same results.

I see this when editing the "multivac" account in 97682c3:
![tupm-edit-account-experiment](https://user-images.githubusercontent.com/915863/54091712-c26adb80-4348-11e9-9b01-7b32bcdf1827.png)

On master, it looks like this:
![tupm-edit-account-master](https://user-images.githubusercontent.com/915863/54091718-d31b5180-4348-11e9-8429-36873510a901.png)

Reply:
Actually, I should reject this PR. I was not content with the result anyways. Besides, a refactor should have *no* functional changes, neither in function nor in layout. So, for some reason, I messed up.

You might want to remove this PR and I will continue on a clean fork.

Reply:
Okay, we can close this PR and consider these changes in a future PR.